Source/RestoredState.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  RestoredState.swift
+//  RxBluetoothKit
+//
+//  Created by Kacper Harasim on 21.05.2016.
+//  Copyright © 2016 Polidea. All rights reserved.
+//
+
+import Foundation
+import CoreBluetooth
+
+/**
+ Convenience class which helps reading state of restored BluetoothManager
+ */
+public struct RestoredState {
+
+    public let restoredStateData: [String:AnyObject]
+
+    let bluetoothManager: BluetoothManager
```

Comment:
Hm, every time user gets access to `RestoredState` instance it by calling `listenOnRestoredState() ` on BluetoothManager. So i think in most common use case it'll not be a problem.